app/controllers/auth_controller.rb

```diff
@@ -10,7 +11,6 @@ def callback
 
     result = sign_in_with_token(
       token,
-      inline_flow: ActiveRecord::Type::Boolean.new.cast(params[:inline_flow]),
```

Comment:
Since this is no longer being used on the server side, we can remove the param from the client side API call, i.e.: https://github.com/chaynHQ/soulmedicine/blob/9100caa343bb1b5e17ce33cbc0be3a440600ed6c/app/webpack/components/authentication/sign_in.vue#L223